Jenkinsfile

```diff
@@ -23,6 +23,12 @@ pipeline {
       }
     }
 
+    stage('Copyright Headers check') {
+      steps {
+        sh 'python /home/TestData/check_copyright_header.py --dir .'
```

Comment:
Should this code be a part of NeMo itself rather than just inside Jenkins?

Reply:
oh wait... this is not part of test suite?

hmmm, I think it might be useful to give the user option to run it locally, similarly as black-based formatting check/fix.

@blisc ?

Reply:
Good question, ultimately, I decided to leave it out as it has nothing to do with NeMo and it hopefully will require no updates in the future

Reply:
Hmm, what tomasz said also has merit.
I could add it to the test suite, but I would also have to copy our entire copyright header into the test suite, which is awkward to me